hasura/metadata/databases/default/tables/public_distributions.yaml

```diff
@@ -2,6 +2,15 @@ table:
   name: distributions
   schema: public
 object_relationships:
+- name: distribution_creator
```

Comment:
i think if you create the foreign key first before adding the relationship, it should look something like that instead

```
object_relationships:
- name: distribution_creator
  using:
    foreign_key_constraint_on: created_by
```

Reply:
thanks @zashton - added the migrations and changed this.